cmd/stork/stork.go

```diff
@@ -588,6 +589,13 @@ func runStork(mgr manager.Manager, ctx context.Context, d volume.Driver, recorde
 				log.Fatalf("Error initializing cluster domain controllers: %v", err)
 			}
 		}
+
+		if c.Bool("webhook-controller") {
+			log.Infof("Creating mutating webhook after leader election")
+			if err := webhookadmission.CreateMutateWebhookRuntime(); err != nil {
+				log.Fatalf("Error creating webhook: %v", err)
+			}
+		}
```

Comment:
This doesn't need to be inside if(d != nil) condition since we are not passing driver like other calls. For consistency lets take it out of the block.

Reply:
Done